ui/scripts/instances.js

```diff
@@ -4189,6 +4189,10 @@
                 allowedActions.push("storageSnapshot");
             }
 
+            if (jsonObj.hypervisor == 'KVM' && jsonObj.pooltype == 'PowerFlex') {
+                allowedActions.push("snapshot");
+            }
+
```

Comment:
@sureshanaparti do we need a Primate change for this or is it already handled there?

Reply:
> @sureshanaparti do we need a Primate change for this or is it already handled there?

@shwstppr Updated this change in primate. PR: https://github.com/apache/cloudstack-primate/pull/863.